src/fat/bios_param.rs

```diff
@@ -39,6 +46,36 @@ impl BIOSParam {
         }
     }
 
+    /// Extract the BIOS Parameter Block (BPB) from the FAT filesystem image.
+    pub fn from(image_fn: String)
```

Comment:
Maybe rename to from_file? Otherwise it might look like you're parsing the string as bios params.
And/or use `std::path::Path`

Reply:
Consulted the Rust guys and they guided me to use AsRef<Path> since it'll accept a String *or* Path (which is pretty neat)